api/app/controllers/api/v1/applications_controller.rb

```diff
@@ -16,7 +16,14 @@ def index
 
         # POST /applications
         def create
+            # if we passed in an id that exists, we want to update
+            if params[:id] && Application.exists?(params[:id])
+                update and return
+            end
+
+            params.require(:applicant_id)
             if invalid_id(Session, :session_id) then return end
+            if invalid_id(Applicant, :applicant_id) then return end
```

Comment:
you can write this as `return if invalid_id(Applicant, :applicant_id)`

Reply:
Okay. I'll do that. Though, do you mind commenting on PR #183 instead, if you're going to review the controller.

Reply:
Done.